docs/ContentTypeParser.md

```diff
@@ -3,6 +3,8 @@
 ## `Content-Type` Parser
 Natively, Fastify only supports `'application/json'` and `'text/plain'` content types. The default charset is `utf-8`. If you need to support different content types, you can use the `addContentTypeParser` API. *The default JSON and/or plain text parser can be changed.*
 
+*Note: If you decide to specify your own content type with the `Content-Type` header, UTF-8 will not be the default. Be sure to include UTF-8 like this `text/html; charset=utf-8`.*
```

Comment:
I would put this sentence in the `Reply.md` doc, on the `type` method
https://github.com/fastify/fastify/blob/main/docs/Reply.md#typecontenttype